docs/unattend.xml

```diff
@@ -16,19 +16,22 @@
          <CEIPEnabled>0</CEIPEnabled>
       </component>
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+         <!-- Windows host computer name -->
          <ComputerName>windows-host</ComputerName>
-         <ProductKey>My_Product_key</ProductKey>
+         <!-- Windows product key -->
+         <ProductKey>_WINDOWS_PRODUCT_KEY_</ProductKey>
```

Comment:
Is there a reason for this change?

Reply:
Yes, to standardize the usage of parameters within the unattend.xml file.

Reply:
Such changes should go into a separate PR for ease of review unless it is a required change. Moreover I [recall](https://github.com/openshift/windows-machine-config-operator/pull/466#discussion_r656400174) this being deleted in a later commit.

Reply:
Agree, will rollback